### PR TITLE
RethinkDB 1.13.1-1

### PR DESCRIPTION
**Pull request by AtnNn**
Thanks.

### CODE REVIEW

**Comment by DomT4** [comment]
Is the ` # boost 1.56 compatibility ` patch still necessary with this new update, and the new version of Boost available?

**Comment by AtnNn** [comment]
Yes. I get build errors when removing the patch.

**Comment by DomT4** [comment]
Cool. Thanks for checking! :+1:

File: Library/Formula/rethinkdb.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class Rethinkdb < Formula
   homepage "http://www.rethinkdb.com/"
-  url "http://download.rethinkdb.com/dist/rethinkdb-1.15.2.tgz"
-  sha1 "31c14c764355e555734c7f4479397bd3bd7e0e44"
+  url "http://download.rethinkdb.com/dist/rethinkdb-1.15.3-1.tgz"
+  version "1.13.1-1"
```
Comment (mikemcquaid) [inline Library/Formula/rethinkdb.rb]: Hmm, why does the version here differ from the tarball name?

Comment (AtnNn) [inline Library/Formula/rethinkdb.rb]: That was a typo on my part.

**Pull request closed without merging**